docs/proposals/label-associated-quota.md

```diff
@@ -0,0 +1,68 @@
+# Label Constrained Quota Allocation
+
+## Problem
+Cluster-administrators want to allot a total quota cap to a particular entity (users to start with) and allow that entity to allocate the quota amongst "his" projects as he sees fit.
+Our mechanism for relating entities is label selection, so we can create a `ResourceQuotaAllocation` object at the cluster scope that applies allotment totals against `ResourceQuota`
+objects.  By associating projects to owners using structured labels, it becomes possible to allocate quota to the projects "owned" by a given user.
+
+
+## Types
+```go
+// ResourceQuotaAllocation sets the maximum amount of resources that a group of projects may use.  It is enforced by preventing individual
+// ResourceQuota allocations to the set of selected projects from exceeding set limits.  If a single project is selected by multiple
+// ResourceQuotaAllocations, then the most restrictive limit wins.
+type ResourceQuotaAllocation struct{
+	unversioned.TypeMeta
+	kapi.ObjectMeta
+
+	Spec ResourceQuotaAllocationSpec
+	Status ResourceQuotaAllocationStatus
+}
+
+type ResourceQuotaAllocationSpec struct{
+	// Selector is used to find the set of projects that this ResourceQuotaAllocation is for
+	Selector extensions.LabelSelector
+
+	// Allocation is the description of the maximum quota sizes allowed across all the selected projects
+	Allocation kapi.ResourceQuotaSpec
+}
+
+type ResourceQuotaAllocationStatus struct{
+	// Allocated is the current state of usage across all selected projects
+	Allocated kapi.ResourceQuotaStatus
+
+	// UsedByProject makes association for users a little easier AND it allows us to update only when needed.
+	UsedByProject NamespacedResourceList
```

Comment:
> A user trying to figure out "which project is taking all my quota" can look at this map to find big consumers.

But the *map* contains exactly one `namespace` because `NamespacedResourceList` isn't a map.